integration/subxt-tests/src/cases/structs.rs

```diff
@@ -0,0 +1,229 @@
+use contract_transcode::ContractMessageTranscoder;
+use hex::FromHex;
+use parity_scale_codec::{Decode, DecodeAll, Encode};
+use rand::Rng;
+use sp_core::{hexdisplay::AsBytesRef, keccak_256, U256};
+
+use crate::{load_project, DeployContract, Execution, ReadContract, WriteContract, API};
+
+#[tokio::test]
+async fn case() -> anyhow::Result<()> {
+    let api = API::new().await?;
+
+    let code = std::fs::read("./contracts/structs.wasm")?;
+
+    let p = load_project("./contracts/structs.contract")?;
```

Comment:
The `.contract` file also contains the wasm. Can we use it from there?

Reply:
Fixed in the latest commit